CHANGELOG.md

```diff
@@ -81,6 +81,11 @@ non-ascii paths while adding files from "Connected file share" (issue #4428)
 - Visibility and ignored information fail to be loaded (MOT dataset format) (<https://github.com/opencv/cvat/pull/5270>)
 - Added force logout on CVAT app start if token is missing (<https://github.com/opencv/cvat/pull/5331>)
 - Missed token with using social account authentication (<https://github.com/opencv/cvat/pull/5344>)
+- Fix chart not being upgradable. In order to upgrade, delete the CVAT Deployments before upgrading
+  (<https://github.com/opencv/cvat/pull/5371>):
```

Comment:
These instructions should be in our documentation somewhere. Could you please add it?